styles/globals.scss

```diff
@@ -26,63 +32,78 @@ $rounded-values: (
     border-radius: map-get($rounded-values, $size);
 }
 
-#__next {
-    font-family: 'Roboto', sans-serif;;
+@mixin flex-row {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+@mixin flex-column {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
 }
```

Comment:
Adding css to center it could be confusing since the mixin name just states that it's flex-column. Either renaming it to something like ```flex-col-center``` and ```flex-row-center``` could work or remove the centering part. Removing the centering part would make atleast one of the mixins pretty useless since it doesn't really save a lot of time/space though or what do you think?